main.yaml

```diff
@@ -1,48 +1,41 @@
 features:
-  - title: What is QMC-HAMM?
+  - title: Dense hydrogen diffusion Monte Carlo (DMC) database
     name: what-wt
-    content: |
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-      commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
-      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
-      est laborum.
+    content:
```

Comment:
I think you might still need the `|` character here, to make it valid YAML.  You can check if it's valid YAML by trying to load it into python, with:

```python
import yaml
yaml.load(open("main.yaml", "r"), Loader=yaml.SafeLoader)
```
and seeing if it throws an error.

Reply:
Oh interesting, thanks. When I was generating the website it did not seem to have issues but I didn't even realize the character was removed. We should leave it in as you suggest.

Reply:
For what it's worth, I was in fact incorrect!  It is not necessary to include the character.